.github/workflows/integration_test_run.yaml

```diff
@@ -200,7 +222,16 @@ jobs:
           fi
           echo "MODULE=$module" >> $GITHUB_ENV
           echo "UNIQUE_ARTIFACT_SUFFIX=$unique_artifact_suffix" >> $GITHUB_ENV
+      - name: Setup Astral UV
+        if: ${{ inputs.with-uv }}
+        uses: astral-sh/setup-uv@v5
```

Comment:
Can you pin this action to a specific version? We aim to have repeatable builds :)

Reply:
Sure, it's currently pinned to `v5`. Should I pin it to a specific commit or the latest version?